src/components/TodoList/TodoList.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import './TodoList.scss';
+import TodoInfo from '../TodoInfo/TodoInfo';
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  todos: Todo[];
+};
+
+const Todolist: React.FC<Props> = ({ todos }) => {
+  return (
+    <ul>
+      {todos.map(todo => (
+        <li className="list-item">
```

Comment:
```suggestion
        <li className="list-item">
```
You've forgotten about `key` attribute